Broken test suite

Every time I run `npm test` on the current main branch, I get:

```sh
/Users/jsumners/Projects/oss/fastify/fastify/test/types/import.js
  1:1   error  Strings must use singlequote  @stylistic/quotes
  1:13  error  Extra semicolon               @stylistic/semi
  2:32  error  Strings must use singlequote  @stylistic/quotes
  2:62  error  Extra semicolon               @stylistic/semi

✖ 4 problems (4 errors, 0 warnings)
  4 errors and 0 warnings potentially fixable with the `--fix` option.
```

But I can't find that file. So, two things:

1. Someone who knows where that file comes from, please provide a way to fix this error.
2. Can we please disable the singlequote rule? It will prevent writing strings with template literals (e.g. when the string needs a singlequote inside of it, it's easier to use a template literal to reduce the need for escaping characters).

--- a/eslint.config.js
+++ b/eslint.config.js
@@ -7,7 +7,8 @@ module.exports = [
     ignores: [
       'lib/configValidator.js',
       'lib/error-serializer.js',
-      'test/same-shape.test.js'
+      'test/same-shape.test.js',
+      'test/types/import.js'
     ]
   },
   ...neo({

--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
     "test:ci": "npm run unit -- --coverage-report=lcovonly && npm run test:typescript",
     "test:report": "npm run lint && npm run unit:report && npm run test:typescript",
     "test:validator:integrity": "npm run build:validation && git diff --quiet --ignore-all-space --ignore-blank-lines --ignore-cr-at-eol lib/error-serializer.js && git diff --quiet --ignore-all-space --ignore-blank-lines --ignore-cr-at-eol lib/configValidator.js",
-    "test:typescript": "tsc test/types/import.ts && tsd",
+    "test:typescript": "tsc test/types/import.ts --noEmit && tsd",
     "test:watch": "npm run unit -- --watch --coverage-report=none --reporter=terse",
     "unit": "tap",
     "unit:junit": "tap-mocha-reporter xunit < out.tap > test/junit-testresults.xml",

